ELK/filebeat.yml

```
@@ -0,0 +1,19 @@
+
+processors:
+  - add_fields:
+      target: ''
+      fields:
+        environment: production
+
+output.elasticsearch:
+  hosts: ["https://192.168.45.179:9200"]
+  api_key: "${ELASTICSEARCH_API_KEY}"
+  ssl:
+    certificate_authorities: ["/usr/share/filebeat/http_ca.crt"]
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)